lib/chef/knife/bootstrap.rb

```diff
@@ -22,229 +22,31 @@
 require "chef/knife/bootstrap/chef_vault_handler"
 require "chef/knife/bootstrap/client_builder"
 require "chef/util/path_helper"
+require "chef/knife/bootstrap/options"
 
 class Chef
   class Knife
     class Bootstrap < Knife
       include DataBagSecretOptions
+      # Command line flags and options for bootstrap - there's a large number of them
+      # so we'll keep this file a little smaller by splitting them out.
+      include Bootstrap::Options
```

Comment:
i'm actually not too wild about this change since the bootstrap options are tightly coupled to the code and this leads to flipping between the two files.  if it isn't actually being reused somewhere it just creates a context switch between the two different files, and there's no sharp separation of concerns.  keeping them in the same file makes it easy to search for a string to determine that an option might (e.g.) be dead code and unused anywhere.